### ISSUE TITLE
Feature description stored twice in XML feature model format (sporadically)

### ISSUE THREAD

**Issue by tthuem** [FeatureIDE/FeatureIDE#661]
#### Prerequisitives
* **FeatureIDE-Version:** v3.3.0

### Issue description
From time to time, descriptions for the very same feature are stored twice in the XML file. This does not always happen and looks like this:

```
<and abstract="true" mandatory="true" name="A">
	<description>
		my description
	</description>
	<description>
		my description
	</description>
```

This is not a big deal with FeatureIDE, as it basically ignores one of them during parsing, but is potentially a problem for other tools parsing the FeatureIDE format. The problem needs to be reproduced in v3.3.0 and then identified what could cause the problem to understand whether we need to fix that for the next release or not.

Who can work on that?

**Comment by Kogoro** [comment]
See #541, which was fixed in 3.3.2.

**Comment by tthuem** [comment]
Thanks for the pointer!
